ocs_ci/framework/conf/default_config.yaml

```diff
@@ -138,3 +140,8 @@ UPGRADE:
 # or ping in ocs infra chat room.
 AUTH:
     test_quay_auth: test_secret
+
+# Defines the configuration of Independent Ceph Cluster
+INDEPENDENT_MODE:
+  admin_keyring:
+     key: 'xyz=='
```

Comment:
why we need another level key ```key```, can't we directly have keyring string for the key ```admin_keyring```

Reply:
we can, i just used full line as template, if we want 'value' only here, I have to update the template, either of those choice looks fine to me.